index.js

```diff
@@ -46,7 +46,7 @@ function getCommonParent(nodes) {
 			});
 		}
 	}
-	return path[path.length - 1];
+	return path ? path[path.length - 1] : document;
```

Comment:
What is this change for?

Reply:
I forget the exact context since it was awhile ago but there was some case where path was undefined which broke the rest of the code path and caused extraneous js errors to show up in the console. I figured if there was no common path then checking the document was as good a defensive option as I had, happy to switch it to something else if you prefer.

Reply:
Throws runtime errors when `path === null`, I see this happen when a dialog is removed.